Update metadata

damacus opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/sous-chefs/line/50)
<!-- Reviewable:end -->

damacus committed:
Update metadata to point to sous-chefs

damacus committed:
Tidy up

tas50 commented on metadata.rb:
```diff
@@ -1,13 +1,13 @@
-issues_url 'https://github.com/someara/line-cookbook/issues' if respond_to?(:issues_url)
-chef_version      '>= 12.5' if respond_to?(:chef_version)
+version      '1.0.0'
+source_url   'https://github.com/sous-chefs/line-cookbook' if respond_to?(:source_url)
```
respond_to isn't necessary on the supermarket metadata. That was in 12.0+

tas50 commented on resources/delete_from_list.rb:
```diff
@@ -78,7 +78,6 @@
       FileUtils.copy_file(temp_file.path, new_resource.path)
       FileUtils.chown(file_owner, file_group, new_resource.path)
       FileUtils.chmod(file_mode, new_resource.path)
-      new_resource.updated_by_last_action(true)
```
You'll want to convert these over to converge_by blocks

damacus committed:
use converge_by blocks to notify resource is converging.

Fix failing regex test
I don't really know if the converge_by blocks are actually correct...

damacus committed:
Update converge_by blocks using @MarkGibbons suggestions.

MarkGibbons commented on resources/add_to_list.rb:
```diff
@@ -35,7 +35,7 @@
       log "Impacted line: #{line}" do
-        level :debug
+        level :info
       end
```
At this point the line matches the regex but it may not be changed.  Log creates output and a changed resource even if nothing else changes.

damacus committed:
Use Chef::Log.info instead of the Log resource so we don't add ourselves
to the resource collection

damacus committed:
disable chefspec tests as they're providing annoingly little value.

Please reenable this is you can fix them!

damacus committed:
Update CHANGELOG
